test/Api.Tests/ControllerTests/AnnotationTagController.TagRelations.Test.cs

```diff
@@ -18,9 +18,6 @@ namespace PaderbornUniversity.SILab.Hip.CmsApi.Tests.ControllerTests
     public class AnnotationTagRelationsControllerTest
     {
         private ControllerTester<AnnotationController> _tester;
-        private User _admin;
-        private User _student;
-        private User _supervisor;
         private AnnotationTag _tag1;
         private AnnotationTag _tag2;
         private AnnotationTag _tag3;
```

Comment:
Now we have the database values stored in two classes (here and in the `ControllerTester`) which is not good. You should remove all these class variables that hold database values from this class and replace all their references with the class variables / properties of the `ControllerTester`